### PR TITLE
fix: update breadcrumb markup for a11y

### LINKED ISSUE #3608: Accessibility: missing aria-label for breadcrumbs
As per title, the aria-label is missing in the breadcrumb

### PR DESCRIPTION
**Pull request by saad-mo**
Closes #3608

### CODE REVIEW

File: projects/storefrontlib/src/cms-components/navigation/breadcrumb/breadcrumb.component.html [hunk 1]
```diff
@@ -1,6 +1,12 @@
-<nav>
-  <span *ngFor="let crumb of crumbs$ | async">
-    <a [routerLink]="crumb.link" [innerHTML]="crumb.label"></a>
-  </span>
+<nav aria-label="Breadcrumb">
```
Comment (Pucek9) [inline projects/storefrontlib/src/cms-components/navigation/breadcrumb/breadcrumb.component.html:1]: https://travis-ci.com/github/SAP/spartacus/jobs/519146902#L1575 😉

File: projects/storefrontlib/src/cms-components/navigation/breadcrumb/breadcrumb.component.html [hunk 2]
```diff
@@ -1,6 +1,12 @@
...
+  <ol>
+    <li span *ngFor="let crumb of crumbs$ | async; last as isLast">
+      <a
+        [routerLink]="crumb.link"
+        [innerHTML]="crumb.label"
+        [attr.aria-current]="isLast ? 'page' : null"
```
Comment (Pucek9) [inline projects/storefrontlib/src/cms-components/navigation/breadcrumb/breadcrumb.component.html:7]: string 'page' also looks as hardcoded. Let's add `aria-current` to list in our `i18n-lint` command

**Review by morganm58 — APPROVED** [review]

File: projects/storefrontlib/src/cms-components/navigation/breadcrumb/breadcrumb.component.html (hunk 1 above)
Comment (developpeurweb) [reply projects/storefrontlib/src/cms-components/navigation/breadcrumb/breadcrumb.component.html:1]: This is works perfect for NVDA and JAWS. Unfortunately, VoiceOver fails to vocalize it, even though it shows up correctly in the rotor. After some research and testing this one below turned out to be a good cross-SR combination:
Screen Shot 2021-06-28 at 1 09 48 PM

**Review by developpeurweb — CHANGES_REQUESTED** [review]

**Comment by saad-mo** [comment]
This PR is stale and includes changes for b2b org which is out of scope for SR efforts for 5.0 will open a new PR

**Pull request closed without merging**